### DISCUSSION TITLE
PCI/IO Bandwidth

### DISCUSSION THREAD

**Discussion by jramasam123** [opcm/pcm#265] [Q&A] [answered]
Hi, I ran pcm and my cpu is:
Intel(R) Xeon(R) W-10885M CPU @ 2.40GHz, cpu model = 165 (I believe it is comet lake). It is a one socket workstation laptop. When I run pcm-pcie, pcm-iio, it says not supported cpu. Is it possible to get IO/PCIe utilization/bandwidth in this system from
PCM OR any other utility (I'm using windows 10 server OS).  I've a PCIe GPU card and I need to know how much data is sent to/from it.
Thanks, Jay

**Comment by opcm** [comment]
I think the closest is the "IO" bandwidth metric in pcm.exe

**Reply by jramasam123** [reply]
Thanks for the prompt response. I guess, I can not use PCM for my need.

**Comment by opcm** [comment, marked as answer]
@jramasam123 
The IO metric is based on the UNC_IMC_DRAM_**IO**_REQUESTS counter documented here:
https://software.intel.com/content/www/us/en/develop/articles/monitoring-integrated-memory-controller-requests-in-the-2nd-3rd-and-4th-generation-intel.html

The traffic from/to the integrated graphics ("[GT](https://en.wikipedia.org/wiki/Intel_Graphics_Technology) engine") is a separate category counted in UNC_IMC_DRAM_**GT**_REQUESTS.

**Reply by opcm** [reply]
the IO metric includes PCIe read/writes to main memory controller but does not include other components of the PCIe traffic (for example read/write from CPU caches).
